code/game/atoms_movable.dm

```diff
@@ -375,18 +375,29 @@
 	loc = new_loc
 	Moved(old_loc)
 
-////////////////////////////////////////
-// Here's where we rewrite how byond handles movement except slightly different
-// To be removed on step_ conversion
-// All this work to prevent a second bump
-/atom/movable/Move(atom/newloc, direction, glide_size_override = 0)
-	. = FALSE
-	if(!newloc || newloc == loc)
-		return
+/atom/movable/Move(atom/newloc, direction, glide_size_override)
+	var/atom/movable/pullee = pulling
+	if(!moving_from_pull)
+		check_pulling()
+	if(!loc || !newloc || loc == newloc)
```

Comment:
this loc == newloc check didn't return altogether.

That might be a behaviour change, althought i don't know when would Move be called with loc == new loc and what it should do in that case.

Insight from someone more experienced would be appreciated